changelog.d/20240508_111720_dave_fix_cors_headers.md

```diff
@@ -0,0 +1 @@
+- [Bugfix] Remove CORS_ALLOW_HEADERS setting from the LMS/Studio config template. This is set separately by LMS and Studio config files in edx-platform and should not be overridden by Tutor, since the header types to allow is application logic and will not vary between sites. This fixes a bug where course import in Studio using the new Course Authoring MFE was broken in Tutor deployments because it required additional headers to be allowed (content-range and content-disposition). This had been set correctly in edx-platform, but was being overridden by Tutor. (by @ormsbee)
```

Comment:
```suggestion
- [Bugfix] Remove CORS_ALLOW_HEADERS setting from the LMS/Studio config template. This setting, which holds site-agnostic application logic, is now consistently set to a reasonable value upstream by LMS and CMS config. Using the upstream values fixes a bug where course import in Studio using the new Course Authoring MFE was broken in Tutor deployments because it required additional headers to be allowed (content-range and content-disposition) (by @ormsbee)
```
Does this looks good @ormsbee ? I can update your commit message to match when I squash-merge.